app/controllers/auth/omniauth_callbacks_controller.rb

```diff
@@ -97,11 +112,20 @@ def github_member?(conf)
   end
 
   # Get user's teams and check if one match to restriction.
-  def member_of(url)
+  def member_of(url, per_page: nil)
     # Get user's groups.
     token = request.env["omniauth.auth"].credentials["token"]
-    resp = Faraday.get url, access_token: token
-    teams = JSON.parse resp.body
+    teams = []
+    # groups are paginated !
```

Comment:
Remove this comment. Instead, let's put this info in the documentation of the method to something like this:

```ruby
# Get user's teams and check if one match to restriction. This method uses pagination, and the
# caller can specify the number of teams per page in the `per_page` parameter.
def member_of(url, per_page: nil)
```